war/src/main/java/io/lumeer/engine/util/Resources.java

```diff
@@ -56,7 +56,11 @@
    @Produces
    @Dependent
    public Logger produceLog(InjectionPoint injectionPoint) {
-      return Logger.getLogger(injectionPoint.getMember().getDeclaringClass().getName());
+      return produceLog(injectionPoint.getMember().getDeclaringClass().getName());
+   }
+
+   public static Logger produceLog(final String className) {
+      return Logger.getLogger(className);
```

Comment:
Logger cannot be directly injected into serializable CDI beans (app., session and conversation scoped). So this is a workaround to ease its creation as some contributors used println instead.